### PR TITLE
Updated instances of OpenShift to {product-title} in Welcome

### PR DESCRIPTION
**Pull request by ahardin-rh**
Skipped Release Notes pages for OSD 3.1 and 3.2, as well as OSE 3.1 and 3.2 for now

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @tnguyen-rh @bfallonf @tpoitras PTAL

This concludes the mass update of OpenShift to {product-title}. As noted, I am skipping the Release Notes content for now, assuming they should be maintained as-is. 

cc @vikram-redhat

File: release_notes/index.adoc [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
 ifdef::openshift-enterprise[]
-The following release notes for OpenShift Enterprise 3.1 and xPaaS summarize all
+The following release notes for {product-title} 3.1 and xPaaS summarize all
 new features, major corrections from the previous version, and any known bugs
 upon general availability.
 endif::[]
 
 ifdef::openshift-dedicated[]
-The following release notes for OpenShift Dedicated, a new OpenShift offering
+The following release notes for {product-title}, a new OpenShift offering
```
Comment (bfallonf) [inline release_notes/index.adoc]: @ahardin-rh Is the one above OK?

Comment (ahardin-rh) [inline release_notes/index.adoc]: @bfallonf yes, I think so because it is within an ifdef statement for Dedicated

**Comment by ahardin-rh** [comment]
No rev history needed

**Pull request merged**